### ISSUE TITLE
stripUrlFileSearchAndHash() adds extra trailing slash

### ISSUE THREAD

**Issue by freshp86** [Polymer/polymer-bundler#562]
```js
stripUrlFileSearchAndHash('chrome://bookmarks/app.html'); // Returns chrome://bookmarks//
```

Note the double slash at the end.  This ends up in the final HTML bundle as

```html
<dom-module id="bookmarks-edit-dialog" assetpath="chrome://bookmarks//">
```

The code in question is at https://github.com/Polymer/polymer-bundler/blob/master/src/url-utils.ts#L46, and it seems that it could be easily fixed by using the [ensureTrailingSlash()](https://github.com/Polymer/polymer-bundler/blob/master/src/url-utils.ts#L31) helper a few lines above.

cc @usergenic

**Comment by usergenic** [comment]
I am fairly certain this is a bug in node's abysmal url parsing library.  Prioritizing fix.

**Comment by freshp86** [comment]
Perhaps the root cause is in node's URL parsing library. FWIW, I was able to fix this locally by changing url-utils.ts line 46 as follows
```js
u.pathname = path.posix.dirname(u.pathname + '_') + '/';
```

```js
u.pathname = ensureTrailingSlash(path.posix.dirname(u.pathname + '_'));
```

**Comment by usergenic** [comment]
Fixed in #564
